joomla/joomla-cms
# [4.0] [com_finder] JS and HTML improvements

### Summary of Changes

This PR fixes the styling for the Smart Search (Filter View) and replaces that lovely jQuery with vanilla JS.

### Testing Instructions

- Apply PR
- Go to Components >> Smart Search >> Filter >> New
- Ensure the styling is ok
- Ensure clicking the Toggle buttons checks/unchecks the checkboxes

Note that the Expand/Collapse button in the top right has a JS error caused by Bootstrap Alpha 6 and will be fixed in Beta 1
